src/Console/InstallCommand.php

```diff
@@ -262,7 +262,7 @@ protected function installLivewireStack()
             $this->removeDarkClasses((new Finder)
                 ->in(resource_path('views'))
                 ->name('*.blade.php')
-                ->notName('welcome.blade.php')
+                ->filter(fn ($file) => $file->getPathname() !== resource_path('views/welcome.blade.php'))
```

Comment:
I couldn't use `notPath` here in the same way as the Inertia stack because the `welcome.blade.php` file we want to exclude is in the root of the directory we're searching, and so anything I passed to `notPath` would also match the `welcome.blade.php` file in the `components` directory.